doc/install_ubuntu.txt

```diff
@@ -144,13 +164,22 @@ system package, you can run this:
 Bleeding edge
 ~~~~~~~~~~~~~
 
-Do like in the section "Updating Theano", but use
-``git+git://github.com/Theano/Theano.git`` instead of ``theano``.
+Change to the Theano directory and run:
+
+.. code-block: bash
+    
+    git pull
+    python setup.py develop --user
```

Comment:
Now there are two "Bleeding edge" sections.  Could you make it clearer that this is to "Update Bleeding Edge" (perhaps by changing the section title to just that).

Also to update a checkout you only need the `git pull` step, the setup.py call is needed once.